spring-boot-project/spring-boot/src/test/java/org/springframework/boot/context/properties/ConfigurationPropertiesBeanTests.java

```diff
@@ -197,6 +197,7 @@ void getWhenHasFactoryMethodAndBeanAnnotationFavorsFactoryMethod() throws Throwa
 	void getWhenHasValidatedBeanBindsWithBeanAnnotation() throws Throwable {
 		get(ValidatedBeanConfiguration.class, "validatedBean", (propertiesBean) -> {
 			Validated validated = propertiesBean.asBindTarget().getAnnotation(Validated.class);
+			assertThat(validated).isNotNull();
```

Comment:
I don't think this is an improvement. It makes the test more verbose and I think the benefit is negligible. Should `validated` be `null`, the NPE that would occur on the next line when calling `validated.value()` will make that clear.

Reply:
It makes IDEA and code analysis tools happy, is it worthy?

Reply:
No, I don't think so.